### DISCUSSION TITLE
0.0.2

### DISCUSSION THREAD

**Discussion by anubrag** [dot-agent/nextpy#87] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/dot-agent/nextpy/releases/tag/'>0.0.2</a>.</em>

**Comment by kumaraguru1735** [comment]
it's the same as Reflex,
Please do something different from Reflex
Please Change Chakra UI, 
On the new update, (tailwind=None) in the config file bug was fixed
I need something different from Reflex  because the compile time takes too much time
